app/static/js/shared_list.js

```diff
@@ -0,0 +1,37 @@
+let items_num = 1;
+
+document.addEventListener('click', function(e) {
+    if (e.target && e.target.id.includes('btnRemoveItem')){
+        removeItem(e.target.id);
+    }
+    else if (e.target && e.target.id.includes("btnRemoveLabel")) {
+        let parent = document.getElementById(e.target.id).parentElement.id
+        removeItem(parent);
+    }
+    else if (e.target && e.target.id == "btnAddItem"){
+        addItem();
+    }
+});
+
+
+function addItem() {
+    let remove_button = 7;
+    let remove_label = 1;
+    let item = document.getElementById("shared_list_item").cloneNode(true);
+    item.style.display = "flex";
```

Comment:
Prefer to change the item's class, and not the dynamically changing the `style` attribute